components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,181 @@
+                            )}
+                            {customMenu && (
+                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
+                            )}
+                        </Typography>
+                    }
+                />
+                {props.children}
+            </Toolbar>
+
+            {menuItems && Boolean(anchorEl) && (
+                <Menu
+                    id="dropdown-toolbar-menu"
+                    data-testid="dropdown-toolbar-menu"
+                    elevation={0}
+                    getContentAnchorEl={null}
+                    anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+                    transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+                    anchorEl={anchorEl}
+                    open={Boolean(anchorEl)}
+                    onClose={(): void => setAnchorEl(null)}
+                    MenuListProps={{ style: { padding: 0 } }}
+                    PaperProps={{
+                        style: {
+                            maxHeight: MENU_ITEM_HEIGHT * 4.5,
+                            width: 200,
+                        },
+                    }}
+                >
+                    {!customMenu &&
+                        menuItems.map((item: ToolbarMenuItem, index: number) => (
+                            <MenuItem
+                                key={`Toolbar-Option-${index}`}
+                                id={`menu-item-${index}`}
+                                data-testid={`menu-item-${index}`}
+                                onClick={(): void => {
+                                    setAnchorEl(null);
+                                    item.onClick();
+                                }}
+                                className={clsx(defaultClasses.menuItem, classes?.menuItem)}
+                            >
+                                {item.label}
+                            </MenuItem>
+                        ))}
+                </Menu>
+            )}
+
+            {customMenu && Boolean(anchorEl) && (
+                <Menu
+                    id="dropdown-toolbar-menu"
+                    data-testid="dropdown-toolbar-menu"
+                    elevation={0}
+                    getContentAnchorEl={null}
+                    anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+                    transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+                    anchorEl={anchorEl}
+                    open={Boolean(anchorEl)}
+                    onClose={(): void => setAnchorEl(null)}
+                    MenuListProps={{ style: { padding: 0 } }}
+                    PaperProps={{
+                        style: {
+                            maxHeight: MENU_ITEM_HEIGHT * 4.5,
+                            width: 200,
```

Comment:
We might not want a fixed width on the dropdown, especially if they are passing in custom content

Reply:
we can set a minWidth though, like we discussed yesterday